contracts/token/RarePizzasBox.sol

```diff
@@ -88,18 +100,19 @@ contract RarePizzasBox is
 
     function purchase() public payable virtual override {
         require(
-            block.timestamp >= publicSaleStart_timestampInS || allowed(msg.sender),
-            "RAREPIZZA: The sale hasn't started yet"
+            block.timestamp >= publicSaleStart_timestampInS ||
```

Comment:
Could we split this up in 2 requires?

Reply:
not necessary to incur the second check if the sale is already on